### PR TITLE
Document SSL Auto-Renewal tool

### PR DESCRIPTION
**Pull request by vanics**
## Work completed:

Add documentation for tool being utilized to auto-renewal ssl certificates for dev and test environments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,12 @@ This application uses both a PostgreSQL database as its main storage as well as
 
 For more information, see [*Database*](docs/database.md).
 
+## SSL Certificates Renewal
+
+SSL Certificates for the Dev and Test environments are automatically renewed using let's encrypt through a tool called openshift-acme.
```
Comment (rhattie-ey) [inline README.md:134]: Improper capitalization.
* "Certificates" should not be capitalized.
* Let's Encrypt is a proper noun and should be capitalized as such.
* "openshift-acme" should potentially be wrapped in backticks to make it clearer as a package name. Less important, though.

File: docs/ssl-certificates-renewal.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# SSL Certificates Renewal Process
+
+SSL Certificates for the Dev and Test environments are automatically renewed using let's encrypt through a tool called openshift-acme.
```
Comment (rhattie-ey) [inline docs/ssl-certificates-renewal.md:3]: Capitalization, as with README.

File: docs/ssl-certificates-renewal.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+The controller was written in Go language and is deployed using an Openshift Deployment script.
+
+The ACME controller/service listens to changes to routes and keep track of certificate lifecyles. It will automatically start a certificate renewal process when required.
+
+As part of the renewal process, Openshift-ACME exposes a new temporary route on Openshift pointing to an auto-generated static file that is used to validate domain name ownership over a specific Openshift route (subdomain name).
+
+### How is Openshift-ACME set up on this project?
+
+Openshift-ACME was deployed using [Single Namespace deployment strategy](https://github.com/tnozicka/openshift-acme/tree/master/deploy#single-namespace).
```
Comment (rhattie-ey) [inline docs/ssl-certificates-renewal.md:23]: Should be "using the" or "using a"

**Review by rhattie-ey — COMMENTED** [review]
Looks good, nice to see this documented! Just needs a couple minor edits.

**Comment by vanics** [comment]
Thank you for catching those, @rhattie-ey!

File: docs/ssl-certificates-renewal.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# SSL Certificates Renewal Process
+# SSL certificates Renewal Process
```
Comment (rhattie-ey) [inline docs/ssl-certificates-renewal.md:1]: Certificates should still be capitalized here. It's a title.

**Review by rhattie-ey — COMMENTED** [review]
One remaining typo, but otherwise looks good!

**Review by rhattie-ey — APPROVED** [review]

**Pull request merged**